rts/lra/lra-coordinator/src/main/java/io/narayana/lra/coordinator/internal/LRARecoveryModule.java

```diff
@@ -122,6 +122,12 @@ private void doRecoverTransaction(Uid recoverUid) {
 
                 if (!inFlight && lra.isRecovering()) {
                     lra.replayPhase2();
+
+                    if (!lra.isRecovering()) {
+                        if (lraService != null) {
```

Comment:
would not be nice to concat the two conditions under one `if` statement?

Reply:
+1